src/cluster.ts

```diff
@@ -693,7 +693,7 @@ Please use the format 'my-cluster' or '${instance.name}/clusters/my-cluster'.`);
    * region_tag:bigtable_api_cluster_set_meta
    */
   setMetadata(
-    metadata: SetClusterMetadataOptions,
+    metadata: BasicClusterConfig,
```

Comment:
This is not a breaking change because `BasicClusterConfig` includes all the parameters that `SetClusterMetadataOptions` includes. We need to do this to allow the user to pass in `location` parameter, which shouldn't be required, but seems to be accepted in other codebases when we update clusters.

Reply:
I believe this will be a compilation error for upstream users because `location` is `require3d` on `BasicClusterConfig`, what I might do instead is add `location` as an optional parameter on `SetClusterMetadataOptions`.

Reply:
Great point. By adding this optional parameter to the interface it also means we make fewer changes too which is good because it keeps this PR small. Done.